src/scss/_icon.scss

```diff
@@ -29,35 +29,15 @@
 .emc-icon.emc-inverted {
   background-color: white;
 }
-.emc-icon-facebook {
-  -webkit-mask-image: url(../icons/iconmonstr-facebook-6.svg);
-  mask-image: url(../icons/iconmonstr-facebook-6.svg);
-}
-.emc-icon-twitter {
-  -webkit-mask-image: url(../icons/iconmonstr-twitter-1.svg);
-  mask-image: url(../icons/iconmonstr-twitter-1.svg);
-}
-.emc-icon-linkedin {
-  -webkit-mask-image: url(../icons/iconmonstr-linkedin-3.svg);
-  mask-image: url(../icons/iconmonstr-linkedin-3.svg);
-}
-.emc-icon-instagram {
-  -webkit-mask-image: url(../icons/iconmonstr-instagram-11.svg);
-  mask-image: url(../icons/iconmonstr-instagram-11.svg);
-}
-.emc-icon-youtube {
-  -webkit-mask-image: url(../icons/iconmonstr-youtube-6.svg);
-  mask-image: url(../icons/iconmonstr-youtube-6.svg);
-}
 .emc-icon-search {
-  -webkit-mask-image: url(../icons/iconmonstr-magnifier-2.svg);
-  mask-image: url(../icons/iconmonstr-magnifier-2.svg);
+  -webkit-mask-image: url(emc-icons-path("magnifier"));
+  mask-image: url(emc-icons-path("magnifier"));
 }
 .emc-icon-close {
-  -webkit-mask-image: url(../icons/close-24px.svg);
-  mask-image: url(../icons/close-24px.svg);
+  -webkit-mask-image: url(emc-icons-path("close"));
+  mask-image: url(emc-icons-path("close"));
 }
 .emc-icon-menu {
-  -webkit-mask-image: url(../icons/menu-24px.svg);
-  mask-image: url(../icons/menu-24px.svg);
+  -webkit-mask-image: url(emc-icons-path("menu"));
+  mask-image: url(emc-icons-path("menu"));
 }
```

Comment:
Aren't these icons already covered by being defined in the icon map?  Why do they need to be defined here?

Reply:
Good catch. Yes I was concentrating on moving the icons over and did not realize that I can remove these now. Let me review the other ones as well.  Thanks!